metrics-operator/controllers/common/providers/dynatrace/dynatrace_dql.go

```diff
@@ -337,3 +348,48 @@ func (d *keptnDynatraceDQLProvider) retrieveDQLResults(ctx context.Context, hand
 	}
 	return result, nil
 }
+
+// extractValueFromRecord extracts the latest value of a record.
```

Comment:
would it make sense to move the following functions in to a separate common file? They seem to be general functionality functions, so it would makes sense to me

Reply:
right now these are specifically for handling payloads from DQL, so at this point i think we do not get much value from making them general